endpoints/k8s/k8s

```diff
@@ -383,7 +383,7 @@ while read line; do
             exit_error "Exiting: exit code 2 (timeout) from roadblock iteration $iter, sample $samp"
         fi
     done
-done < "$rickshaw_run_dir/client-server-bench-commands/client-1"
+done < "$rickshaw_run_dir/bench-cmds/client/1"
```

Comment:
The hardcoding of "1" seems curious here.  I guess I missed this in the previous set of K8S patches.  Why is it hardcoded?

Reply:
It is hardcoded as 1 because it does not matter which client's file it uses as long as the file is there.  Client "1" is guaranteed to be there.

Reply:
ack, makes sense